### PR TITLE
feat(policies): move policies examples to policy_yaml

### PR DESCRIPTION
**Pull request by Automaat**
Fixes: #2020

### CODE REVIEW

File: app/_src/policies/meshcircuitbreaker.md [hunk 1]
```diff
@@ -474,7 +474,35 @@ spec:
 
 #### Basic circuit breaker for outbound traffic from web, to backend service
 
-{% policy_yaml usage use_meshservice=true %}
+{% if_version lte:2.8.x %}
+{% policy_yaml usage-28x use_meshservice=true %}
```
Comment (lahabana) [inline app/_src/policies/meshcircuitbreaker.md:478]: If this is lte 2.8 shoudn't it not be using meshservice?

**Review by lahabana — APPROVED** [review]
Thank you so much!

File: app/_src/policies/meshcircuitbreaker.md (hunk 1 above)
Comment (Automaat) [reply app/_src/policies/meshcircuitbreaker.md]: removed

File: app/_src/policies/meshhttproute.md [hunk 2]
```diff
@@ -462,15 +478,17 @@ spec:
                 requestMirror:
                   percentage: 30
                   backendRef:
-                    kind: MeshServiceSubset
-                    name: backend_kuma-demo_svc_3001
-                    tags:
-                      version: v1_experimental
+                    kind: MeshService
```
Comment (Automaat) [inline app/_src/policies/meshhttproute.md:481]: I am not sure what to do with this, the example generated from it when not using MeshService in incorrect. There is the same example in MeshTCPRoute. For old scenario with `kuma.io/service` we should use MeshServiceSubset and with tags

Comment (slonka) [reply app/_src/policies/meshhttproute.md:481]: We should fix the generator code right? Can we if_version it for now, create an issue to fix it and merge this?

**Review by slonka — APPROVED** [review]

File: app/_src/policies/meshhttproute.md (hunk 2 above)
Comment (slonka) [reply app/_src/policies/meshhttproute.md:481]: oh, it's in `backendRef` - maybe @lobkovilya has an idea how to fix it

Comment (slonka) [reply app/_src/policies/meshhttproute.md:481]: fixed this on https://github.com/kumahq/kuma-website/pull/2056

Comment (Automaat) [reply app/_src/policies/meshhttproute.md:481]: it was fixed with https://github.com/kumahq/kuma-website/pull/2056 I will update examples

**Pull request merged**